### DISCUSSION TITLE
Why is light basically infinite in a universe that is basically finite?

### DISCUSSION THREAD

**Discussion by mikehelland** [a-cosmology-group/acg#202] [Metaphysics and Philosophy]
This isn't a scientific argument, so I thought Metaphysics and Philosophy was a good spot for this.

Light has an infinite range.

Why? The universe has a finite past, and future that involves heat death. Seems like it's over-engineered for no reason. 

I talk about that in this short video:

https://www.youtube.com/watch?v=A2hf6k0SytU

**Comment by RedshiftDrift** [comment]
1- The concept of light as a race car is incompatible with quantum mechanics and special relativity.  When the race car leaves A nobody knows if it will make it to the finish line.  However, light is an interaction between A and B.  The photon connects A and B in a causal manner: once the photon has left A, nothing can stop it from reaching B.  In special relativity the distance bridged by light is neither time-like, nor space-like, nor infinite, but actually null.

'**c**' is for the Latin celeritas (= 'celerity').  Even if it has units of [distance/time], it's not a speed but a fundamental property of energetic interactions between material particles distributed in space-time $\displaystyle c = \sqrt{\frac{E}{m}}$.

2- Better cosmologies don't have a time boundary nor a space boundary.

With a better cosmology, a quantum description, and special relativity, the concerns expressed in the video simply don't exist.

**Reply by HanDeBruijn** [reply]
No. Light is an electromagnetic  **wave** that is travelling with a finite speed :
[Visualizing video at the speed of light — one trillion frames per second](https://www.youtube.com/watch?v=EtsXgODHMWk) .

**Reply by mikehelland** [reply]
I think we all agree that light is not a car and, since we are in this forum, that the standard model of cosmology needs a replacement.

I just thought it was odd there is something infinite _inside_ something that is practically finite.

Not sure if other people had mentioned that, or how anyone feels about it.

**Reply by Pierre314159** [reply]
@mikehelland No, we don't "all agree". As I wrote in an other thread, I don't think the "standard model" of cosmology "needs a replacement". I think it needs to be thrown away, ditched, annihilated, forgotten... The very idea of a "standard model" should be abandoned. It is not science.

So, take me out of "all".

**Reply by RedshiftDrift** [reply]
@HanDeBruijn your example is a classical demonstration, which of course propagates at $c$.  I'm talking about the quantum properties of light that are much faster.  As far as we can see experimentally, the interaction is instantaneous.
See [https://blogs.scientificamerican.com/news-blog/quantum-weirdnes-wins-again-entangl-2008-08-13/](https://blogs.scientificamerican.com/news-blog/quantum-weirdnes-wins-again-entangl-2008-08-13/) and a more recent measurement 
[https://www.nature.com/articles/nature11472](https://www.nature.com/articles/nature11472)

**Reply by RedshiftDrift** [reply]
@mikehelland To continue on the philosophical aspect, there is no *inside* of the universe since the universe has no bounds.  When you write *inside* it implies an *outside*.  By definition there is nothing outside the universe because the universe is everything.

**Reply by HanDeBruijn** [reply]
@RedshiftDrift .

According to some cosmological models, there is an **observable** universe, which is finite and _inside_ the "whole" universe, while the rest of the universe is _outside_ that observable universe of ours and it is infinite.

**Reply by mikehelland** [reply]
@RedshiftDrift 

Fair point. I was thinking of "the universe" as more of an abstract container for time, space, matter, etc, than a specific physical volume.

**Comment by budrap00** [comment]
You are both correct. Louis is describing the behavior of light from the 4-dimensional perspective of light and Han from the perspective of a 3-dimensional observer.
